acceptance/tests/options/config_file/command_line_override.rb

```diff
@@ -0,0 +1,39 @@
+test_name "flags set on the command line override config file settings"
```

Comment:
i'm sure the other tests didn't do this but `test_name` and `step` should have do/end blocks.  this allows beaker to indent the logs properly. this mode is deprecated, if not officially.